notes/fintech-incrementalism-and-responsible-innovation.md

```diff
@@ -2,31 +2,45 @@
 title: Fintech Incrementalism and Responsible Innovation
```

Comment:
My one concern here is that this deep dive covers several topics. I wonder if it would be better to leave this as a pure deep dive and create a distinct claims page with just summary and evaluation and then link to this page for more background - especially if we are splitting out the payment rails stuff.

Reply:
Did a full claims + evaluation breakdown in coggle: https://coggle.it/diagram/YyGhGOfA2wMrF6cQ/t/web3-claim-analysis-for-blockchain-fintech-incrementalism Conclusion is that ultimately this deep dive covers several distinct claims and better to have a claim page for each that references this page (rather than make this into a claim).